.github/workflows/publish-to-pypi.yml

```
@@ -0,0 +1,35 @@
+name: Publish to PyPI
+
+on: workflow_dispatch
+
+jobs:
+  authorize:
+    name: Authorize
+    runs-on: ubuntu-latest
+    steps:
+      - name: ${{ github.actor }} permission check to do a release
+        uses: "lannonbr/repo-permission-check-action@2.0.2"
+        with:
+          permission: "write"
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build-n-publish:
+    name: Build and publish to PyPI
+    runs-on: ubuntu-latest
+    needs: [authorize]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.6
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.6
+      - name: Install dependencies
```

```suggestion
      - uses: actions/checkout@v3

      - name: Set up Python 3.6
        uses: actions/setup-python@v3
        with:
          python-version: 3.6

      - name: Install dependencies
```
nit: I think it is best practice to sepearte steps with a empty line for clarity.
https://github.com/actions/starter-workflows/blob/main/.github/workflows/validate-data.yaml